posts/month-in-mathlib-may-2024.md

```diff
@@ -0,0 +1,184 @@
+  * Joël Riou
+    * [PR #12713](https://github.com/leanprover-community/mathlib4/pull/12713) :: feat(Algebra/Category/ModuleCat): the category of presheaves of modules has colimits
+* Mitchell Lee laid the groundwork for future material on Coxeter groups. [PR #11465] defines the length function on a Coxeter group and proves its basic properties. [PR #11466] defines reflections and inversions in a Coxeter group and proves that the inversion sequence of a reduced word consists of distinct reflections.
+* In [PR #12647], Mitchell Lee formalized a proof of the equational criterion for vanishing, a necessary and sufficient criterion for a sum of pure tensors in the tensor product of two modules to vanish.
+  * Oliver Nash
+    * [PR #13208](https://github.com/leanprover-community/mathlib4/pull/13208) :: feat: equality of coroots implies equality of roots for Lie algebras
+    * [PR #13298](https://github.com/leanprover-community/mathlib4/pull/13298) :: feat(Algebra/Lie/Weights): `n` such that `n • α + β` is a weight is consecutive.
+    * [PR #13217](https://github.com/leanprover-community/mathlib4/pull/13217) :: feat(Algebra/Lie/Semisimple): API for semisimple Lie algebras
+    * [PR #13370](https://github.com/leanprover-community/mathlib4/pull/13370) :: chore(Algebra/Lie/Abelian): a nonabelian atomic Lie ideal is perfect
+    * [PR #13387](https://github.com/leanprover-community/mathlib4/pull/13387) :: chore(Algebra/Lie/Submodule): the lattice of Lie submodules is atomic
+    * [PR #13391](https://github.com/leanprover-community/mathlib4/pull/13391) :: feat(Algebra/Lie/InvariantForm): Lie algebras with a nondegenerate invariant form are semisimple
+    * [PR #13265](https://github.com/leanprover-community/mathlib4/pull/13265) :: feat(Algebra/Lie): Killing Lie algebras are semisimple
+    * [PR #13179](https://github.com/leanprover-community/mathlib4/pull/13179) :: feat: add lemma `LieAlgebra.IsKilling.span_weight_isNonZero_eq_top`
+    * [PR #12712](https://github.com/leanprover-community/mathlib4/pull/12712) :: feat: Cartan subalgebras contain only semisimple elements if the Killing form is non-singular and the coefficients are a perfect field.
+    * [PR #13076](https://github.com/leanprover-community/mathlib4/pull/13076) :: feat: define `sl₂` triples and prove basic result
```

Comment:
This can all go if we accept my suggestion [above](https://github.com/leanprover-community/blog/pull/76/files#r1659906136)
```suggestion
```